test/models/apple/episode_test.rb

```diff
@@ -156,4 +135,17 @@
       end
     end
   end
+
+  describe "#synced_with_apple?" do
+    let(:apple_episode_api_response) { build(:apple_episode_api_response, publishing_state: "PUBLISH") }
+
+    it "should be false when drafting" do
```

Comment:
So is this the only real change in the tests?  The rest of this looks like factory-shuffling.  (And I only see the one changed line outside the tests).

Reply:
The `build(:uploaded_apple_episode)` factory here precipitated a lot of the changes, but yep, mostly trying to get the api responses well represented in the factories to build up that happy path/uploaded episode. And that meant adding a few factories